### ISSUE TITLE
Optional Ruby Langauge?

### ISSUE THREAD

**Issue by JHero23** [NITSkmOS/Algorithms#549]
I have a question, is it ok if we can use Ruby programming language for our algorithms and data structure commits. I use Ruby programming language as well. Also will it be valid for our hacktoberfest commits?

**Comment by sangamcse** [comment]
Yes. For that, someone needs to add configurations for Travis and coala.

**Comment by sangamcse** [comment]
Any commit that gets merged in this repo is counted as hacktoberfest commit.
